pkg/kubelet/kuberuntime/fake_kuberuntime_manager.go

```diff
@@ -39,7 +39,7 @@ import (
 )
 
 const (
-	fakeSeccompProfileRoot = "/fakeSeccompProfileRoot"
+	fakeRootDirectory = "/fakeroot"
```

Comment:
does this make unit tests actually write to `/fakeroot` files on disk?

Reply:
No, none of the tests in the kube runtime manager write data to that directory. It's mainly used for seccomp localhost profile path matching: https://github.com/kubernetes/kubernetes/blob/583ac1463d6389e882ce2a3b9bcecb667df54705/pkg/kubelet/kuberuntime/helpers_test.go#L34-L36

Reply:
ok... trying really hard to avoid regressing @BenTheElder's "unit tests should not require root" progress (https://github.com/kubernetes/kubernetes/issues/99881)